aioseop_class.php

```diff
@@ -4480,10 +4480,12 @@ function get_prev_next_links( $post = null ) {
 				}
 			}
 			if ( ! empty( $prev ) ) {
-				$prev = $this->substr( $prev, 9, - 2 );
+				$tmp_str = $this->substr( $prev, ( $this->strpos( $prev, 'href="' ) + 6 ) );
```

Comment:
@EkoJR 

I don't think _wp_link_page should be used as it is a private function. https://developer.wordpress.org/reference/functions/_wp_link_page/. 

I also don't think substr is the best way to do this. Maybe a more resilient method would be to use http://php.net/manual/en/class.domdocument.php

Reply:
@contactashish13 It is not possible to use `wp_link_page()`. The function is for rendering purposes and has no control over the page number.
https://developer.wordpress.org/reference/functions/wp_link_pages/

I did however implement DOMDocument as requested, but between the two it does add more processing which would make it less efficient. Just a note to be mindful of with large processing times.

Reply:
I agree with @EkoJR about processing time. @wpsmort maybe this can be tested with a large site?

Reply:
@EkoJR maybe add a TODO to not use a private function that we can revisit later in case it cannot be fixed now?